### PR TITLE
Set up a dummy marketing page

### PR DESCRIPTION
**Pull request by barto74**
Issue #124 
Its a very very simple page, but it at least gives something when new people end up on our domain.

### CODE REVIEW

**Comment by bfirsh** [comment]
Nice!

The way it has been wired up will break a few things I think. (E.g. logging in won’t take you anywhere, home screen app will always land on the marketing page, etc.)

A better solution would probably be to make `/` show different things depending on whether you are logged in or not. I’ll wire that up by pushing directly to this branch. (Nobody merge the pull request until that is done please!)

File: src/pages/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+body {
```
Comment (bfirsh) [inline src/pages/HomePage.css]: All the CSS here should be contained within `.HomePage` to stop it from colliding with other things on the site. If you do this, you also need to add a `className` below.

An alternative solution would be to use the CSS in JavaScript technique like in some of the other components. That automatically stops collisions. I am open to either method, though. Using plain CSS is more approachable somehow.

File: src/pages/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./HomePage.css";
+import PropTypes from "prop-types";
+import Grid from "@material-ui/core/Grid";
+
+const HomePage = ({ user }) => {
+  return (
+    <Grid container xs={12} sm={12} md={7} lg={6} xl={5} className="grid">
```
Comment (bfirsh) [inline src/pages/HomePage.js]: Replace `className=“grid”` with `className=“HomePage”`, if you are wrapping up the CSS above.

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./HomePage.css";
+import PropTypes from "prop-types";
+import Grid from "@material-ui/core/Grid";
+
+const HomePage = ({ user }) => {
```
Comment (bfirsh) [inline src/pages/HomePage.js]: `{ user }` can be removed here - it is not passed by anything.

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+          rel="noopener noreferrer"
+        >
+          Github
+        </a>!
+      </div>
+    </Grid>
+  );
+};
+
+HomePage.propTypes = {
```
Comment (bfirsh) [inline src/pages/HomePage.js]: propTypes define the properties that are passed to the component (like arguments to a function). User isn’t passed by anything, so these 3 lines can be removed entirely. This is probably causing an error in your console saying the user property hasn’t been passed.

**Comment by barto74** [comment]
I was aiming at reusing '/' as this marketing dummy, renaming the old  '/' to  '/dashboard'. But apparently i didnt work that one out well enough.

**Comment by bfirsh** [comment]
Yes, lots of things depend on the dashboard being at `/` unfortunately. :)

That is also something we can detangle, but I would argue `/` changing behaviour depending on whether you're logged or not is actually a better user experience. It means if you're logged in, typing "boxwise.co" in your browser will take you straight to your dashboard.

**Comment by barto74** [comment]
Is that desirable behaviour? So I can't go to the marketing page without logging out?
I was just expecting to always see the marketing page at / and depending on the user status either show a login/signup button or a 'logged in as Bart' text.
I see advantages in both sides, but this was what I was expecting.

**Comment by bfirsh** [comment]
Yeah, there are advantages and disadvantages either way. Most sites work with `/` changing behaviour - GitHub, Facebook, Twitter, to name a few off the top of my head.

I would argue the advantage of being taken straight to the app if you type in the URL outweighs the disadvantage of having to log out to see the marketing page (which you probably don't need to see if you're already using the app).

**Comment by barto74** [comment]
It makes sense @bfirsh. Are you fixing this issue according to that approach?

**Comment by bfirsh** [comment]
Yeah, but I'm not going to be able to get to it for a couple of days. You can fix the issues I mentioned in the meantime, if you'd like.

**Comment by HaGuesto** [comment]
@bfirsh @barto74 
I agree that you should be directly refered to the dashboard if you are already a user and logged in.

File: src/pages/HomePage.js (hunk 2 above)
Comment (HaGuesto) [reply src/pages/HomePage.js]: xs sm md lg xl attributes can only be placed in grid items.

**Pull request closed without merging**